src/routes/messages/entities/schemas/__tests__/create-message.dto.schema.spec.ts

```diff
@@ -7,8 +7,9 @@ import { ZodError } from 'zod';
 describe('CreateMessageDtoSchema', () => {
   describe('message', () => {
     it('should validate a valid record message', () => {
+      const message: Record<string, unknown> = JSON.parse(fakeJson());
```

Comment:
Nit: I'd cast this as we aren't "sure"
```suggestion
      const message = JSON.parse(fakeJson()) as Record<string, unknown>;
```